components/list/list-item-mixin.js

```diff
@@ -530,6 +586,17 @@ export const ListItemMixin = superclass => class extends LocalizeCoreElement(Lis
 		this._hovering = false;
 	}
 
+	_onNestedSlotChange() {
+		if (this.selectable) {
+			this._onNestedSlotChangeCheckboxMixin();
+		}
+		const nestedList = this._getNestedList();
+		if (this._hasChildren !== !!nestedList) {
```

Comment:
Not intending to bike-shed, but I'm wondering if this should be `_hasNestedList`. Thoughts? Would it matter if there was a nested list that's empty? It's been really difficult in the list components, but we've tried to be consistent and accurate with how stuff is named, hopefully to avoid any confusion.

Reply:
I'd be fine renaming to `_hasNestedList`, not a lot of effort to clean that up.